src/main/java/uk/gov/hmcts/cmc/claimstore/config/properties/notifications/EmailTemplates.java

```diff
@@ -111,4 +111,7 @@
 
     @NotEmpty
     private String defendantAdmissionResponseToClaimant;
+
+    @NotEmpty
+    private String claimantRejectedPartAdmitOrStatesPaidEmailToDefendant;
```

Comment:
As there is no different email template for these, do we really need to say what type of defendant's response claimant is rejecting in name of this field? May be just  `claimantRejectedEmailToDefendant`

Reply:
As the wording on the GOV Notify template is very specific I have done this. The fear is that if they change the template to be more specific to States Paid/Part Admit as opposed to the general format it is at the moment. I agree with 'claimantRejectedEmailToDefendant' if the template wasn't specific.